cmd/mistry/main.go

```diff
@@ -74,11 +74,12 @@ JOB PARAMETERS:
 EXAMPLES:
 	1. Schedule a job with a group and some parameters and put artifacts under
 		/tmp/yarn using rsync. Prefixing a file name with @ will cause the contents
-		of yarn.lock to be sent as parameters.
+		of yarn.lock to be sent as parameters. Any dynamic options prepended with a '_'
```

Comment:
To be consistent, let's call them parameters (or params) instead of dynamic options (see: https://github.com/skroutz/mistry/wiki/Glossary).

Also, `... will be treated as opaque for the result cache, but will be available during the build.` The build cache is the `/data/cache/` directory which is persisted between builds of the same group. The JobID hashing logic is called the result cache.